server/src/main/java/org/opensearch/identity/noop/NoopTokenHandler.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.identity.noop;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.opensearch.OpenSearchException;
+import org.opensearch.identity.IdentityService;
+import org.opensearch.identity.tokens.AuthToken;
+import org.opensearch.identity.tokens.NoopToken;
+import org.opensearch.identity.tokens.TokenManager;
+
+/**
+ * This class represents a Noop Token Manager
+ */
+public class NoopTokenHandler implements TokenManager {
+
+    private static final Logger log = LogManager.getLogger(IdentityService.class);
+
+    /**
+     * Issue a new Noop Token
+     * @return a new Noop Token
+     */
+    @Override
+    public AuthToken issueToken() {
+        return new NoopToken();
```

Comment:
We probably don't need NoopToken at all: 

```suggestion
        return new AuthToken() {};
```

Or even have constant for it 

```
private static final AuthToken NOOP_TOKEN = new AuthToken() {};
```